aliBuild

```diff
@@ -16,6 +16,10 @@ from alibuild_helpers.utilities import format, getVersion, detectArch
 from alibuild_helpers.analytics import decideAnalytics, askForAnalytics, report_screenview, report_exception, report_event
 from alibuild_helpers.analytics import enable_analytics, disable_analytics
 import traceback
+if float('%d.%d' % sys.version_info[:2]) < 2.7:
```

Comment:
Ok, fair enough. I will adapt it and make sure it works on slc5. Thank you for your contribution!